src/boot/hwids/device4.json

```diff
@@ -0,0 +1,9 @@
+{
+    "type": "firmware",
+    "name": "Device 4",
+    "compatible": "test,vmware",
```

Comment:
should be changed to `"fwid" : "...."`

Reply:
Yes when you change this, it invites lots of more work in ukify.py.

Reply:
I will try to fix this somehow.